### PR TITLE
clarify language in ai_models.md

### PR DESCRIPTION
**Pull request by discdiver**
Minor edits to ai_models docs.

One confusing thing still is using plugins for AI models - how do you make them available if they aren't available by default?

"You can customize the LLM's temperature, or give your AI Model access to Wikipedia, 
internal documentation, or a sanitized executable environment by using Marvin's [plugins](plugins.md). AI models have no plugins available by default in order to minimize the possibility of confusing behavior. See [AI function docs](ai_functions.md). "

### CODE REVIEW

File: docs/guide/concepts/ai_models.md [hunk 1]
```diff
@@ -43,17 +43,19 @@ Resume('Ford Prefect • (555) 5124-5242 • ford@prefect.io').json(indent = 2)
...
+The `ai_model` decorator can be applied to any Pydantic model. For optimal results, the model should have a descriptive name, annotated fields, and a class docstring. The model does not need to have any pre-processing or post-processing methods written. However, advanced users can add these methods to influence the output.
 
-When an `ai_model`-decorated model is instantiated with unstructured text, all available information is sent to the AI, which generates a predicted output. This output is parsed according to the model's schema and returned as the model's instance.
+When an `ai_model`-decorated model is instantiated with unstructured text, all available information is sent to the AI. Note that the data is sent to the LLM as the first `root_validator`. The AI generates a predicted output that is parsed according to the model's schema and returned as the model's instance.
```
Comment (zzstoatzz) [inline docs/guide/concepts/ai_models.md:58]:
```suggestion
When an `ai_model`-decorated class is instantiated with unstructured text, all available information is sent to the AI. The AI generates a predicted output that is parsed according to the model's schema and returned as an instance of the model.
```

**Review by zzstoatzz — CHANGES_REQUESTED** [review]

**Review by zzstoatzz — APPROVED** [review]
thanks!

**Comment by discdiver** [comment]
Thank you @zzstoatzz ! 

Do you want to add language to clarify this section of the doc:
context: One confusing thing still is using plugins for AI models - how do you make them available if they aren't available by default?

"You can customize the LLM's temperature, or give your AI Model access to Wikipedia,
internal documentation, or a sanitized executable environment by using Marvin's [plugins](https://github.com/PrefectHQ/marvin/pull/plugins.md). AI models have no plugins available by default in order to minimize the possibility of confusing behavior. See [AI function docs](https://github.com/PrefectHQ/marvin/pull/ai_functions.md). "

**Comment by zzstoatzz** [comment]
> Do you want to add language to clarify this section of the doc: context: One confusing thing still is using plugins for AI models - how do you make them available if they aren't available by default?

The same way as `ai_fn` - you pass `plugins` to the decorator like [this](https://gist.github.com/zzstoatzz/a16da0594afc2bb751428907e4c2a3cd)

**Pull request merged**